zerver/webhooks/patreon/fixtures/members_pledge_create.json

```diff
@@ -0,0 +1,384 @@
+                "thanks_embed": null,
+                "thanks_msg": null,
+                "thanks_video_url": null,
+                "url": "https://www.patreon.com/zulip"
+            },
+            "id": "1194966",
+            "relationships": {
+                "creator": {
+                    "data": {
+                        "id": "7498102",
+                        "type": "user"
+                    },
+                    "links": {
+                        "related": "https://www.patreon.com/api/user/7498102"
+                    }
+                },
+                "goals": {
+                    "data": []
+                },
+                "rewards": {
+                    "data": [
+                        {
+                            "id": "-1",
+                            "type": "reward"
+                        },
+                        {
+                            "id": "0",
+                            "type": "reward"
+                        },
+                        {
+                            "id": "7027346",
+                            "type": "reward"
+                        },
+                        {
+                            "id": "7027366",
+                            "type": "reward"
+                        },
+                        {
+                            "id": "7027376",
+                            "type": "reward"
+                        },
+                        {
+                            "id": "7027398",
+                            "type": "reward"
+                        }
+                    ]
+                }
+            },
+            "type": "campaign"
+        },
+        {
+            "attributes": {
+                "about": null,
+                "apple_id": null,
+                "can_see_nsfw": null,
+                "created": "2017-08-29T23:33:28.000+00:00",
+                "default_country_code": null,
+                "discord_id": null,
+                "email": "patreon@zulip.com",
+                "facebook": null,
+                "facebook_id": null,
+                "first_name": "Zulip",
+                "full_name": "Zulip",
```

Comment:
It appears we have a name for the user; in these fixtures; we should display it in the notifications.